scalameter-core/src/main/scala/org/scalameter/MeasureBuilder.scala

```diff
@@ -38,12 +43,12 @@ class MeasureBuilder[T, U](
   
       if (ctx(Key.exec.requireGC)) compat.Platform.collectGarbage()
   
-      val measurements = measurer.measure[T, U](
+      val measurements = measurer.measure[T](
         ctx, ctx(Key.exec.benchRuns),
         setup, teardown, regen, b
       )
   
-      resultFunction(measurements)
+      resultFunction(measurements.map(_.value))
```

Comment:
I guess we can do it that way. I don't foresee a need for the aggregator to have access to units, so it could work out.

I'd still be happier if we knew the reason for the error. But serialization stack traces are always fun to deal with..